packages/README.md

```diff
@@ -32,10 +33,39 @@ When creating a new package, you need to provide at least the following:
     	},
     	"publishConfig": {
     		"access": "public"
-    	}
+    	},
+    	// Include this line to include the package as a WordPress script.
+    	"wpScript": true,
+    	// Include this line to include the package as a WordPress script module.
+    	"wpScriptModuleExports": "./build-module/index.js"
     }
     ```
+
     This assumes that your code is located in the `src` folder and will be transpiled with `Babel`.
+
+    For packages that should ship as a WordPress script, include `wpScript: true` in the `package.json` file. This tells the build system to bundle the package for use as a WordPress script.
+
+    For packages that should ship as a WordPress script module, include a `wpScriptModuleExports` field the `package.json` file. The value of this field can be a string to expose a single script module, or an object with a [shape like the standard `exports` object](https://nodejs.org/docs/latest-v20.x/api/packages.html#subpath-exports) to expose multiple script modules from a single package:
```

Comment:
Isn't this a duplication of the `exports` field for regular npm packages. Do we really need to invent our own convention?

Reply:
@sirreal can provide a more exhaustive response. We were discussing using the official configuration for ES Modules:
- [`type`](https://nodejs.org/api/packages.html#type) set to `module`
- [`exports`](https://nodejs.org/api/packages.html#exports) covering entry points

It'll eventually happen, but more work is necessary. For example, all import statements that contain file paths must contain the file extension, for example: `import './file.js';`. That isn't the case currently.